src/server/status_server/mod.rs

```diff
@@ -408,47 +420,143 @@ impl StatusServer {
         )
     }
 
+    pub fn dump_region_meta(
+        req: Request<Body>,
+        router: Option<&RaftRouter<RocksEngine>>,
+    ) -> Box<dyn Future<Item = Response<Body>, Error = hyper::Error> + Send> {
+        lazy_static! {
+            static ref REGION: Regex = Regex::new(r"/region/(?P<id>\d+)").unwrap();
+        }
+
+        let err_resp =
+            |status_code, body| Box::new(ok(StatusServer::err_response(status_code, body)));
+
+        let not_found = || {
+            err_resp(
+                StatusCode::NOT_FOUND,
+                format!("{} not found", req.uri().path()),
+            )
+        };
+
+        let router = match router {
+            None => {
+                return err_resp(
+                    StatusCode::INTERNAL_SERVER_ERROR,
+                    "router not set".to_string(),
+                )
+            }
+            Some(r) => r,
+        };
+
+        match REGION.captures(req.uri().path()) {
+            None => not_found(),
+            Some(cap) => {
+                let id: u64 = match cap["id"].parse() {
+                    Ok(id) => id,
+                    Err(err) => {
+                        return err_resp(
+                            StatusCode::BAD_REQUEST,
+                            format!("invalid region id: {}", err),
+                        );
+                    }
+                };
+                let (tx, rx) = oneshot::channel();
+                match CasualRouter::send(
+                    router,
+                    id,
+                    CasualMessage::Test(Box::new(move |peer| {
```

Comment:
Could we rename `CasualMessage::Test` to a proper name like `CasualMessage::DumpState`? since it's no longer a test only message.

Reply:
`CasualMessage::Test` is renamed to `CasualMessage::AccessPeer`.